common/djangoapps/static_replace/models.py

```diff
@@ -0,0 +1,25 @@
+"""
+Models for static_replace
+"""
+
+from django.db.models.fields import TextField
+from config_models.models import ConfigurationModel
+
+
+class AssetBaseUrlConfig(ConfigurationModel):
+    """Configuration for the base URL used for static assets."""
+    base_url = TextField(
+        blank=True,
+        help_text="The alternative hostname to serve static assets from.  Should be in the form of hostname[:port]."
+    )
+
```

Comment:
It could, but we have no way to signal that back to whatever caching layer ends up getting used.  Since this just controls how links are generated, not how the contentserver middleware behaves, I don't think it would make sense here specifically.